globals/game.gd

```diff
@@ -28,18 +28,17 @@ func set_mode(p_mode):
 func mouse_enter(obj):
 	var text
 	var tt = obj.get_tooltip()
-	if current_tool != null:
-		text = "use.combine_id"
-		if current_action != "":
-			text = tr(current_action + ".combine_id")
-		text = text.replace("%2", tr(tt))
-		text = text.replace("%1", tr(current_tool.get_tooltip()))
-	elif obj.inventory:
+
+	#TODO: Fix inventory combination
+	#if current_action == "pick_up" && text != null:
+		#text = tr(current_action + ".combine_id")
+		#text = text.replace("%2", tr(tt))
+		#text = text.replace("%1", tr(current_tool.get_tooltip()))
+	if obj.inventory:
```

Comment:
Or maybe it doesn't; I'm not sure. I think I made some changes there to support two-click interaction on inventory items. Inventory combine works with this change, but using an inventory item on a non-inventory item seems not to. This also removes localization support, but we haven't done anything to support that yet anyway. Let's just keep this in mind and give it a proper look when we need to.